### ISSUE TITLE
Content ID is duplicated in show partial

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1068]
Violation: ID attribute must be unique.

There should only be one instance of` id="content"`

https://github.com/pulibrary/orangelight/blob/2ae997a121f8a11cdbeffee843fe4ca306aba883/app/views/catalog/show.html.erb
